### DISCUSSION TITLE
Entering a biome gate outpost with insufficient reputation and gaining enough reputation at said outpost doesn't unlock the path to the next biome

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#15359] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

My crew docked to a Coalition colony at the edge of Cold Caverns with 29 reputation. At the outpost, I completed an event which gave us extra Coalition reputation. Enough to meet the biome gate requirement. The person who was supposed to unlock the gate for us did not acknowledge this and demanded a bribe, until we undocked and redocked.
I'm marking this bug as happening "just once" and being multiplayer-specific, because that's the first time it happened to me. But I suspect that it can happen consistently in any mode. The conditions required for it are just rare.

![602960_20241213001404_1](https://github.com/user-attachments/assets/09142e90-8653-41aa-bd5b-41a418a10a91)
![602960_20241213001448_1](https://github.com/user-attachments/assets/8e5fa116-d59e-417e-95f8-62af2aeada82)

### Reproduction steps

1. Enter a biome gate outpost with insufficient reputation.
2. Gain enough reputation to meet the gate requirement while at the outpost. (Can happen through outpost events in regular gameplay.)
3. Approach the person responsible for providing access to the next biome.
4. They treat you as if you don't have enough reputation.

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in our private repo now, fix coming up in a future unstable build and release.
